pkg/manifest/manifest_test.go

```diff
@@ -50,11 +51,13 @@ func TestLoadManifest(t *testing.T) {
 func TestLoadManifest_DeprecatedFields(t *testing.T) {
 	cxt := context.NewTestContext(t)
 
-	cxt.AddTestFile("testdata/porter-with-image.yaml", config.Name)
+	cxt.AddTestFile("testdata/porter-with-deprecated-fields.yaml", config.Name)
 
 	m, err := LoadManifestFrom(cxt.Context, config.Name)
 	require.NoError(t, err, "expected no error")
 	require.NotNil(t, m, "manifest was nil")
+
+	require.Equal(t, "getporter/porter-hello:v0.1.0", m.BundleTag, "manifest has incorrect bundle tag")
```

Comment:
We should be asserting off of the new field Reference, not BundleTag now. After a manifest is loaded, `Reference` should contain the final full reference, and bundletag shouldn't have been modified.